tests/system/python/conftest.py

```diff
@@ -27,12 +32,161 @@ def reset_and_start_foglamp():
     assert "FogLAMP not running." not in stat.stdout.decode("utf-8")
 
 
+def find(pattern, path):
+    result = None
+    for root, dirs, files in os.walk(path):
+        for name in files:
+            if fnmatch.fnmatch(name, pattern):
+                result = os.path.join(root, name)
+    return result
+
+
+@pytest.fixture
+def start_south():
+    def _start_foglamp_south(south_plugin, foglamp_url, config=None, use_pip_cache=True):
+        """Start south service"""
+        _config = config if config is not None else {}
+        pip_cache = "" if use_pip_cache else "--no-cache-dir"
+        data = {"name": "play", "type": "South", "plugin": "{}".format(south_plugin), "enabled": "true",
```

Comment:
service_name also be dynamic, atleast add extra param to _start_foglamp_south `service_name='play'`